### ISSUE TITLE
Turn "expand our donor base" ideas into clear actionable items/issues

### ISSUE THREAD

**Issue by laurenluz** [Giveth/giveth-planning#1099]
@c0ric0ri I know you already started tackling some of this feedback, but would love to break down others into clear ideas.. maybe we make an epic with the ones we/you filter out.

Here's the notion: https://www.notion.so/giveth/GIVpower-Launch-Retrospective-438068375d5a40d488655b5383f11116

**Comment by clara-zi** [comment]
Starting organising some of the ideas into categories - we could use this file as a base to think about priority order and how we will measure success and achievements of each activity.

https://docs.google.com/spreadsheets/d/1DKadtHgIWHldRj0yZSM29oKe0Ll45wut/edit#gid=458351912

**Comment by ahmadabugosh** [comment]
Hey @clara-zi , I added 4 points as well that I can support with (and that I have experience with)  
1) Onboarding flow / improve funnel 
2) Content marketing
3) Pad ads
4) Webinars / meetups

**Comment by laurenluz** [comment]
With the new strategy, we should revisit this... it seems to have fallen through the cracks as well. @c0ric0ri

**Comment by c0ric0ri** [comment]
Hey @laurenluz, yes indeed many of these points were considered in the [marketing strategy outline](https://www.notion.so/giveth/Marketing-Growth-Chapter-Strategy-adf8be2315644047a58f56f1124d0270). I've also linked this spreadsheet there for easy reference.

Other things have already been worked on or started (creating social media templates for projects, onboarding landing pages, starting influencer outreach).

@oyealmond I've included this issue in the list for your reference
